Orange/statistics/util.py

```diff
@@ -344,31 +344,22 @@ def stats(X, weights=None, compute_variance=False):
 
     if X.size and is_numeric:
         if is_sparse:
+            nans = countnans(X, axis=0)
```

Comment:
Did not measure, but wouldn't 't it be better to count nans it columns after `tocsc`? Or, if this does not changes anything, we could remove `tocsc` altogether.

Reply:
I moved it there because `countnans` wants csr:

```
        arr = arr.tocsr()
        counts = _count_nans_per_row_sparse(arr, weights, dtype=dtype)
```

And I kept conversion to csc because `nanmean` calls `mean_variance_axis` which is smart wrt combining axis and sparse matrix type.

Reply:
Smart! Thanks.